benchmarks/views/user.py

```diff
@@ -315,6 +330,34 @@ def instance_has_files(namelist, instances):
     return True, files_list, None
 
 
+def extract_identifiers(zip_ref):
+    # define patterns for each plugin type (data and metrics to be added later)
+    possible_plugins = ["models", "benchmarks"]
+    registry_patterns = {
+        "models": re.compile(r"model_registry\['(.+?)'\]"),
+        "benchmarks": re.compile(r"benchmark_registry\['(.+?)'\]"),
```

Comment:
Won't these also match commented lines? E.g. # model_registry["alexnet"] = .... I would update this to only include non-commented lines.

Reply:
Good catch! Should be fixed now so both inline and own line comments are ignored.